src/views/open_order_list_view.js

```diff
@@ -0,0 +1,167 @@
+import Backbone from 'backbone';
+import OpenOrderView from '../views/open_order_view'
+
+import OpenOrder from '../models/open_order';
+
+const OpenOrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.bus, 'quote_symbols', this.symbolDropdown);
+    this.listenTo(this.bus, 'quote_change', this.checkOpenOrders);
+
+    // list for current quote list
+    this.listenTo(this.bus, 'current_quote_list', this.getQuoteList);
+  },
+  getQuoteList(quoteList) {
+    this.quoteList = quoteList;
+    console.log('the quote list !!')
+    console.log(this.quoteList);
+
+  },
+  render() {
+    this.$('#orders').empty();
+
+    this.model.each((openOrder) => {
+      const openOrderView = new OpenOrderView({
+        model: openOrder,
+        template: this.template,
+        tagName: 'li',
+        className: 'order',
+        bus: this.bus,
+      });
+      // render returns the taskview (this) which allows you to append
+      this.$('#orders').append(openOrderView.render().$el);
+    });
+    return this;
+  },
+  checkOpenOrders(quote) {
+
+    let openOrders = this.model.where({symbol: quote.get('symbol')});
+    if (openOrders.length >= 1) {
+      openOrders.forEach((openOrder) => {
```

Comment:
You don't need to wrap the call to `forEach` with a check that `length >= 1`. If the list has zero elements, then the loop body will execute zero times.